app-admin/syslog-ng/syslog-ng-4.3.1.ebuild

```diff
@@ -21,7 +21,7 @@ RESTRICT="!test? ( test )"
 RDEPEND="
 	>=dev-libs/glib-2.10.1:2
 	>=dev-libs/ivykis-0.42.4
-	>=dev-libs/libpcre-6.1
+	>=dev-libs/libpcre2-10.0
```

Comment:
```suggestion
	>=dev-libs/libpcre2-10.0:=
```

Reply:
Fwiw the subslot on linpcre2 is only for one of its installed libraries so need to check if using that

Reply:
If it only links with pcre2-8, then please don't.

Reply:
> If it only links with pcre2-8, then please don't.

Ok. I updated the patch and removed the subscription to the subslot.

Reply:
> Fwiw the subslot on linpcre2 is only for one of its installed libraries so need to check if using that

TIL. Thanks for pointing this out. And it's even documented in libpcre2's metadata.xml. \o/